circle.yml

```diff
@@ -45,6 +45,9 @@ checkout:
 
 dependencies:
   pre:
+    # CircleCI v1 caching doesn't work for parallel builds
+    # Clean any possible cache from the previous runs to avoid re-using wrong distro cache
+    - rm -rf node_modules
```

Comment:
CircleCI in a "good intentions" caches by default `node_modules` from the `0`th parallel container https://circleci.com/docs/1.0/how-cache-works/#caching-in-parallel-builds and re-using that in other containers.
It means that all 4 different distros reusing that dir and linking to a wrong shared libraries.


It's horrible, like `EL6` links to `U14` libs which means wrong packages and random surprising issues during package installs.